archive/s/scala/README.md

```diff
@@ -5,6 +5,7 @@ Welcome to Sample Programs in Scala!
 ## Sample Programs
 
 - [Hello World in Scala](https://therenegadecoder.com/code/hello-world-in-scala/)
```

Comment:
Just a general request, but could you convert all the links in this file to the format you've started (i.e. links numbered at the end of the file)? While you're there, can you alphabetize the program list? That would be awesome!